ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/onboarding/(step)/1/page.tsx

```diff
@@ -1,3 +1,14 @@
+import StepsPageHeader from '@inngest/components/Steps/StepsPageHeader';
```

Comment:
Could all of these templates be merged into a single page.tsx with a route param? ex. 
```
.../env/[environmentSlug]/onboarding/(step)/[stepSlug]/page.tsx
```

If we have the external content config and components we can have one page right?